docs/advanced-usage/eloquent-casting.md

```diff
@@ -99,14 +99,14 @@ $cdRecord->config; // CdRecordConfig object
 $vinylRecord->config; // VinylRecordConfig object
 ```
 
-When a data object class is abstract and used as an Eloquent cast then this feature will work out of the box.
+When a data object class is abstract and used as an Eloquent cast, then this feature will work out of the box.
 
-The child data object value of the model will be stored in the database as a JSON string with the class name as the key:
+The child data object value of the model will be stored in the database as a JSON string with the class name and the data object properties:
 
 ```json
 {
     "type": "\\App\\Data\\CdRecordConfig",
-    "value": {
+    "data": {
```

Comment:
I looked at the source code and it seems the `data object` value will be stored under the `data` key and not the `value`.
Please correct/revert this if I am wrong.